packages/vaadin-themable-mixin/register-styles.d.ts

```diff
@@ -1,11 +1 @@
-import { CSSResultGroup } from 'lit';
-
-export { css, unsafeCSS } from 'lit';
-
-/**
- * Registers CSS styles for a component type. Make sure to register the styles before
- * the first instance of a component of the type is attached to DOM.
- */
-declare function registerStyles(themeFor: string | null, styles: CSSResultGroup, options?: object | null): void;
-
-export { registerStyles };
+export { registerStyles, css, unsafeCSS } from './vaadin-themable-mixin';
```

Comment:
Let's use `.js` extension for consistency.

```suggestion
export { registerStyles, css, unsafeCSS } from './vaadin-themable-mixin.js';
```